app/views/products/index.html.haml

```diff
@@ -22,4 +22,4 @@
             .form-group
               %button.btn.btn-success{ ng: { click:"addToCart(product)" }}
                 Add To Cart
-              %input.input-sm{ type: "number", ng: { model: "product.quantity" }}
\ No newline at end of file
+              = select_tag('', options_for_select((1..10)), {'ng-model' => "product.quantity", 'ng-init' => '' })
```

Comment:
could also do the same thing  with...
```%input.input-sm{ type: "number", min: 1, max: 10, ng: { model: "product.quantity" }} ```
unless you're dead set on using select tags